backend/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.8'
+services:
+  db:
+    container_name: postgresdb
+    image: postgres
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: potter_db_api_development
+    ports:
+      - "5432:5432"
+    networks:
+      - mynetwork
+    restart: always
+
+  web:
+    build: .
+    container_name: railsapp
+    ports:
+      - "3000:3000"
+    depends_on:
+      - db
+    networks:
+      - mynetwork
+    command: ["sh", "-c", "rails db:migrate && rails db:seed && rails server -b 0.0.0.0"]
```

Comment:
So if we remove it from here ,the readme should contain how to get into the bash of the container and manually start it, since rails db:migrate is needed before rails server @danielschuster-muc

Reply:
> So if we remove it from here ,the readme should contain how to get into the bash of the container and manually start it, since rails db:migrate is needed before rails server @danielschuster-muc

I think you should be able to run `docker-compose exec web rails <any rails cmd>`. Or with rails console etc.